srml/support/src/dispatch.rs

```diff
@@ -1754,10 +1783,15 @@ mod tests {
 
 	#[test]
 	fn weight_should_attach_to_call_enum() {
-		// max weight. not dependent on input.
-		assert_eq!(Call::<TraitImpl>::weighted().weight(100), 3 * 1024 * 1024);
+		// Operational tx.
+		assert_eq!(Call::<TraitImpl>::security_tx().weight(100), 0);
+		assert_eq!(Call::<TraitImpl>::security_tx().priority(100), u64::max_value());
+		// one simply does not return false of the transaction type is operational!.
```

Comment:
```suggestion
		// Operational transactions are never denied no matter how full
		// the block is.
```

Reply:
fair enough. I am personally pretty much okay with having some humor/informal statements as long as it is in test code :p